app/src/main/java/com/example/weatherapp/network/Network.kt

```diff
@@ -0,0 +1,26 @@
+package com.example.weatherapp.network
+
+import com.example.weatherapp.network.service.WeatherService
+import okhttp3.OkHttpClient
+import okhttp3.logging.HttpLoggingInterceptor
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+
+class Network {
+
+    private val service: WeatherService
+    private val baseUrl = "https://www.metaweather.com/api/location/"
+
+    init {
+        val interceptor = HttpLoggingInterceptor()
```

Comment:
Nedostaje ti postavljanje level-a kako bi se logiranje izvrsavalo:

```suggestion
        val interceptor = HttpLoggingInterceptor()
        interceptor.level = HttpLoggingInterceptor.Level.BASIC
```